### PR TITLE
Enable logging for all tests

### PR DESCRIPTION
**Pull request by orestisfl**
### Summary of your changes
- Only enables it when go test -v is used
- Remove repetitive setup in test suites
- Uses test name for logger name

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -24,22 +24,15 @@ import (
...
 
 type ConfigTestSuite struct {
 	suite.Suite
-
-	log *logp.Logger
 }
 
 func TestConfigTestSuite(t *testing.T) {
 	s := new(ConfigTestSuite)
-	s.log = logp.NewLogger("cloudbeat_config_test_suite")
```
Comment (uri-weisman) [inline config/config_test.go:39]: Should be replaced with helper func?

File: magefile.go [hunk 2]
```diff
@@ -337,7 +338,16 @@ func PythonEnv() error {
 	return err
 }
 
-func BuildOpaBundle() (err error) {
+func BuildOpaBundle() error {
+	cspDirectory := os.Getenv(cspDirectoryEnv)
```
Comment (uri-weisman) [inline magefile.go:342]: It is safer to use `os.LookupEnv`

File: processor/add_cluster_id/add_cluster_id_test.go [hunk 3]
```diff
@@ -21,24 +21,16 @@ import (
...
 
 type AddClusterIdTestSuite struct {
 	suite.Suite
-
-	log *logp.Logger
 }
 
 func TestAddClusterIdTestSuite(t *testing.T) {
 	s := new(AddClusterIdTestSuite)
-	s.log = logp.NewLogger("cloudbeat_add_cluster_id_test_suite")
```
Comment (uri-weisman) [inline processor/add_cluster_id/add_cluster_id_test.go:37]: Should we use the helper func?

File: resources/providers/awslib/config_provider_test.go [hunk 4]
```diff
@@ -22,24 +22,16 @@ import (
...
 
 type ConfigProviderTestSuite struct {
 	suite.Suite
-
-	log *logp.Logger
 }
 
 func TestEcrFactoryTestSuite(t *testing.T) {
 	s := new(ConfigProviderTestSuite)
-	s.log = logp.NewLogger("cloudbeat_config_provider_test_suite")
```
Comment (uri-weisman) [inline resources/providers/awslib/config_provider_test.go:38]: same

File: beater/validator_test.go [hunk 5]
```diff
@@ -21,26 +21,19 @@ import (
...
 type ValidatorTestSuite struct {
 	suite.Suite
 
-	log *logp.Logger
 	sut *validator
 }
 
 func TestValidatorTestSuite(t *testing.T) {
 	s := new(ValidatorTestSuite)
-	s.log = logp.NewLogger("cloudbeat_validator_test_suite")
```
[the remark this answers is not in the document]
Comment (orestisfl) [reply beater/validator_test.go:37]: Not used

File: config/config_test.go (hunk 1 above)
Comment (orestisfl) [reply config/config_test.go:39]: Not used

File: magefile.go (hunk 2 above)
Comment (orestisfl) [reply magefile.go:342]: Sorry, accidentally commited this

File: processor/add_cluster_id/add_cluster_id_test.go (hunk 3 above)
Comment (orestisfl) [reply processor/add_cluster_id/add_cluster_id_test.go:37]: `s.log` is not used

File: resources/providers/awslib/config_provider_test.go (hunk 4 above)
Comment (orestisfl) [reply resources/providers/awslib/config_provider_test.go:38]: same :)

**Pull request merged**